src/dist/replication/test/simple_kv/case-211.act

```diff
@@ -70,14 +70,12 @@ wait:on_task_begin:node=r3,task_code=RPC_GROUP_CHECK
 state:{{r1,pri,2,15},{r2,sec,2,15},{r3,pot,2,0}}
 wait:on_rpc_call:rpc_name=RPC_LEARN,from=r3,to=r1
 inject:on_rpc_reply:rpc_name=RPC_LEARN_ACK,from=r1,to=r3
-state:{{r1,pri,2,15},{r2,sec,2,15},{r3,err,2,0}}
```

Comment:
This act fix has been done before (i.e., https://github.com/imzhenyu/rDSN/blob/master/src/dist/replication/test/simple_kv/case-211.act) , and it is due to the optimization in learning process. I'm afraid it seems that the code merge misses some previous changes. You may need to check again about commit https://github.com/imzhenyu/rDSN/commit/0c0869cb42d2d1d56e99fe599e6fb814694a004a#diff-31589f3618bd776324342418b1911d49 to see any accidental changes that cause missed changes.